### ISSUE TITLE
Attempting to install the 2.2.0 version results in "External dependencies not satisfied"

### ISSUE THREAD

**Issue by SigmaMaleOrderBride** [infinitymdm/heroic-games-solus#3]
Hi! First of all, awesome that you're making an .eopkg package of Heroic!

I saw the new release and attempted to update the previous version (which installed and worked flawlessly for me). This however presented me with an error telling me "External dependencies not satisfied". I tried to remove the previous version as well and do a clean install, but I get the same dependency issue. 

I'm not super well versed in package managers, and less so in eopkg, so I'm not entirely sure how I determine what the dependency I'm missing is or if this is something needed to be done on a package level? Hopefully you have a better clue than I!

Including the debug output below.
```
DEBUG: HistoryDB initialized in 0.000891923904419.
DEBUG: A = ['heroic-games-launcher-2.2.0-1-1-x86_64.eopkg']
DEBUG: InstallDB initialized in 0.00305008888245.
DEBUG: PackageDB initialized in 0.0476050376892.
DEBUG: RepoDB initialized in 0.000159978866577.
System error. Program terminated.
<type 'exceptions.Exception'>: External dependencies not satisfied: expat release >= 23
Please use 'eopkg help' for general help.

Traceback:
  File "/usr/bin/eopkg", line 82, in <module>
    cli.run_command()
  File "/usr/lib/python2.7/site-packages/pisi/cli/pisicli.py", line 142, in run_command
    self.command.run()
  File "/usr/lib/python2.7/site-packages/pisi/cli/install.py", line 104, in run
    pisi.api.install(packages, ctx.get_option('reinstall') or reinstall)
  File "/usr/lib/python2.7/site-packages/pisi/api.py", line 69, in wrapper
    ret = func(*__args,**__kw)
  File "/usr/lib/python2.7/site-packages/pisi/api.py", line 471, in install
    return pisi.operations.install.install_pkg_files(packages, reinstall)
  File "/usr/lib/python2.7/site-packages/pisi/operations/install.py", line 201, in install_pkg_files
    raise Exception(_('External dependencies not satisfied: %s') % dep)
```

**Comment by infinitymdm** [comment]
Hi @ItsJimJams ,

I can't say for sure, but the problem here is likely that the release is built against the Solus unstable repository. I assume you're installing on the stable "shannon" repo (you can check using `eopkg lr`).

I'm glad you've called this issue to my attention. I hadn't considered this issue at all, and I should have. I see a couple of potential paths forward:

Option A: I can update the release to a .eopkg built against stable. This is less than ideal for me, as it means I have to test both the stable and unstable versions. This likely means I'd be publishing an untested release, and I don't like that a whole lot for obvious reasons. I could be convinced to go this route, but I'd want to start marking my releases as untested.

Option B: You can wait for my work to be included in the official Solus repository. You can track the progress [here](https://dev.getsol.us/D12883). I'm hoping it will be approved for submission prior to Friday, which would mean it would be included in the Friday sync. You wouldn't have to download my release from Github, as it would be available in the software center or the command line via eopkg. It would also get automatic updates through the software center and all that jazz.
When it does get officially included, I'll probably still publish builds here just in case something goes funny with the Solus infrastructure.

Option C: You can build the package yourself using [the instructions linked in the README](https://github.com/infinitymdm/heroic-games-solus#building-from-source). 

Option D: You can switch to unstable. I don't recommend this unless you specifically want to help out with testing upcoming Solus releases.

I'm open to feedback on which course of action you think I should take.

**Comment by infinitymdm** [comment]
Perhaps an even better option than the ones I listed above is for you to use the v2.1.1 release. The only downside there is that it doesn't have GOG.

**Comment by ItsJimJams** [comment]
I see! In that case I'll just wait until it's included in the official Solus repo and continue to use the 2.1.1 release in the meantime, or if it takes longer than expected I'll just fall back on the AppImage version.

Thanks either way!

**Comment by infinitymdm** [comment]
I've been giving this some more thought, and I think I'm going to start publishing both stable and unstable versions when I do releases. That will begin with today's 2.2.1 patch. You should see that release here in a few hours if everything goes well.

Note that stable releases will be untested, but at least they will be available. Once this is included in the official repo I'll go back to only publishing unstable.

**Comment by infinitymdm** [comment]
@ItsJimJams I've just published the 2.2.1 release including a stable build. Let me know if that works for you.

**Comment by ItsJimJams** [comment]
Yes, that works! Thanks for taking the time :)
